firmware/common/portapack_persistent_memory.cpp

```diff
@@ -252,6 +252,25 @@ struct ui_config_t {
     }
 };
 
+/* Additional UI config.
+ * NB: Will be default init - override in defaults(). */
+struct ui_config2_t {
```

Comment:
I really think we should do this for all the packed bit fields. We could delete a lot of code in this file.

Reply:
We should make read and write but globals instead of repeating it for each new var, but yes

Reply:
I'm not sure I follow. We would still keep the getter/setter functions in the header, we just wouldn't need the internal bit-handling getter/setters as well.

Reply:
Really, I could move all these into the main data_t and reuse the 8 bits saved by removing hide_converter. 
I'll probably leave this here just so we can have a home for additional UI flags that we may add later.